relational/table.py

```diff
@@ -318,22 +318,33 @@ def _gen_units(self, row, proj):
             row[self.index(self.NET_NUM_UNITS_BMR_DATA)] = source
         else:
             dbi_net = _get_dbi_units(proj)
+            ppts_net = proj.field('market_rate_units_net', PPTS.NAME)
             net = dbi_net
-            if dbi_net is not None:
+            # PTS may have an explicitly set 0 unit count for projects
+            # that have no business dealing with housing (possible with
+            # permit type 3 in particular), so we only emit a 0-count
+            # PTS unit count if we had an explicit non-0 PPTS unit
+            # count (therefore indicating a housing-related project that
```

Comment:
Curious more than anything else, are these 'rehab' projects that show up like this typically?

Reply:
i only looked at a few, but they were like very minor things. they had very little other information in the other tables, so they were basically just noisy zeros.

Reply:
sg, probably the best approach that we can take here in that case